client/src/index.jsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
-import { render } from 'react-dom';
-import App from './App';
+import React from "react";
+import { render } from "react-dom";
+import { Provider } from 'react-redux';
+import { store } from './state/store';
+import App from "./App";
 
-const root = document.createElement('div');
-root.setAttribute('id', 'root');
+const root = document.createElement("div");
+root.setAttribute("id", "root");
 document.body.appendChild(root);
 
-render(<App />, root);
+render(
+  <Provider store={store}>
+    <App />
+  </Provider>,
```

Comment:
oh interesting so provider is just a native Redux component to provide the store data?